.openapi-generator-ignore

```diff
@@ -27,3 +27,7 @@ build.gradle
 src/main/java/org/openapitools/client/model/FilterType.java
 README.md
 settings.gradle
+
+#the api client contains the custom code for static initialization from the json file
+src/main/java/io/tiledb/cloud/rest_api/ApiClient.java
```

Comment:
I don't think we should override generated classes. Can we take the approach we had in TileDB-Cloud-Py in that we create higher level classes that wrap the generated API?

Reply:
Created the TileDBClient class to wrap the autogenerated one